addons/website/controllers/main.py

```diff
@@ -660,6 +660,20 @@ def get_new_page_templates(self, **kw):
                 'title': group_el.text,
```

Comment:
Commit message:
- First commit:
  -  `to mark pages available as new page template`  -> templates
  - `section of the new page dialog.` -> Nitpick but "New Page dialog" (since it is the name of the dialog) 

- Second commit:
  - `Because of this such content remains empty.` -> comma after `this`
  - `This commit temporarily adds in info alert` -> an info alert